sdk/core/azure_core/src/task/standard_spawn.rs

```diff
@@ -1,61 +1,119 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-use super::{SpawnHandle, TaskFuture, TaskSpawner};
-use async_trait::async_trait;
+use super::{SpawnedTask, TaskFuture, TaskSpawner};
 #[cfg(not(target_arch = "wasm32"))]
 use futures::executor::LocalPool;
 #[cfg(not(target_arch = "wasm32"))]
 use futures::task::SpawnExt;
 #[cfg(not(target_arch = "wasm32"))]
+use futures::FutureExt;
+#[cfg(not(target_arch = "wasm32"))]
+use std::future::Future;
+#[cfg(not(target_arch = "wasm32"))]
+use std::pin::Pin;
+#[cfg(not(target_arch = "wasm32"))]
+use std::sync::{Arc, Mutex};
+#[cfg(not(target_arch = "wasm32"))]
+use std::task::Waker;
+#[cfg(not(target_arch = "wasm32"))]
+use std::task::{Context, Poll};
+#[cfg(not(target_arch = "wasm32"))]
 use std::thread;
```

Comment:
This is really hard to read. Could you combine all the imports? That's typical. rust-analyzer should do it automatically for non-conditional, but then just manually move stuff in manually.